kitsune/sumo/static/sumo/less/main.less

```diff
@@ -55,73 +56,113 @@ body {
 
     > header .logo {
       float: right;
+      padding-right: 30px;
     }
 
     &.scroll-header {
       > header {
         .logo {
-          left: auto;
-          right: 0;
+          padding-right: 10px;
         }
 
         #support-search {
           left: 0;
+          padding-left: 10px;
           right: auto;
+
+          .searchbox {
+            padding-left: 36px;
+          }
+
+          .submit-button {
+            left: 18px;
+          }
         }
 
         #aux-nav {
-          margin-left: 302px;
-          margin-right: 0;
+          margin-left: 220px;
         }
 
+        #aux-nav-right {
+          margin-left: 240px;
+        }
+      }
+    }
+
+    .search-form-large {
+      .searchbox {
+        padding-left: 60px;
+      }
+      .submit-button {
+        left: 140px;
       }
     }
   }
 
   > header {
-    background: #fff;
-    border-bottom: 1px solid #b3b8bc;
-    border-top: 2px solid #fff;
-    margin-bottom: 20px;
-    padding-bottom: 20px;
+    background: #000;
+    min-height: 100px;
+    margin: 0;
+    padding: 0;
+    position: relative;
+    width: 100%;
+
+    #aux-nav {
+      padding: 29px 0 29px 20px;
+    }
+
+    #aux-nav-right {
+      padding: 29px 20px 29px 0;
+    }
+
+    #support-search {
+      display: none;
+    }
 
     .logo {
+      background: url('../img/mozilla-support-white.svg') top left no-repeat;
+      background-size: 112px 48px;
       float: left;
-      height: 69px;
-      width: 160px;
-      display: block;
-      background: url('../img/mozilla-support.png') 0 0 no-repeat;
-      background-size: 160px 69px;
+      margin-left: 30px;
+      position: relative;
+      
+      a {
+        display: block;
+        height: 48px;
+        width: 112px;
+      }
     }
   }
 
   &.scroll-header {
-    padding-top: 151px;
+    padding-top: 100px;
 
     > header {
-      background: #fff;
+      background: #000;
       box-shadow: 0 0 5px rgba(0, 0, 0, 0.2);
-      height: 60px;
-      padding: 0;
+      min-height: 50px;
+      max-height: 50px;
```

Comment:
If I use a single `height` rule it doesn't work because the header has `min-height: 100px;` when it's not in a scrolling state. The reason we have that `min-height` is when the screen gets smaller the nav links can wrap and not mess with the buttons on the right.